docs/advanced.rst

```diff
@@ -568,8 +568,8 @@ to integrate common database interfaces such as the DB-API specified in
 <http://sqlalchemy.readthedocs.org>`_ or `SQLObject
 <https://pypi.python.org/pypi/SQLObject/>`_.
 
-You will find `here <https://bitbucket.org/Lawouach/cherrypy-recipes/src/tip/web/database/sql_alchemy/>`_
-a recipe on how integrating SQLAlchemy using a mix of
+You will find a recipie at `SQL_Alchemy <https://bitbucket.org/Lawouach/cherrypy-recipes/src/tip/web/database/sql_alchemy/>`_
```

Comment:
This line looks too long. Also, there's a typo. Let's split it.
Also, SQLAlchemy repeated twice is not a good style.
```suggestion
You will find a recipe at `cherrypy-recipes
 <https://bitbucket.org/Lawouach/cherrypy-recipes/src/tip/web/database/sql_alchemy/>`_
```

Reply:
@jay1723 plz click on "Commit suggestion" above ^